src/avro_ocf.erl

```diff
@@ -142,37 +142,54 @@ append_file(Fd, Header, SchemaStore, Schema, Objects) ->
 make_header(Type) ->
   make_header(Type, _ExtraMeta = []).
 
-%% @doc Make ocf header, and append the given extra metadata fields.
--spec make_header(avro_type(), extra_meta()) -> header().
-make_header(Type, ExtraMeta0) ->
-  ExtraMeta = validate_extra_meta(ExtraMeta0),
+%% @doc Make ocf header, and append the given metadata fields.
+%% You can use &gt;&gt;"avro.codec"&lt;&lt; metadata field to choose what data
```

Comment:
quote code in ``'`. e.g. `<<"null">>'
this is more clear to read in code editor.
and will be translated to `<code>&lt;&lt;"null"&gt;&gt;</code>` in html.